.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -32,6 +32,7 @@ NuGet package version:
 | Windows 10 version                  | Saw the problem? |
 | :--------------------------------- | :-------------------- |
 | Insider Build (xxxxx)              | <!-- Yes/No? -->   |
+| November 2019 Update (18363)       | <!-- Yes/No? -->   |
```

Comment:
The new sdk is probably not published yet, I saw a 19041 preview [here](https://www.microsoft.com/en-us/software-download/windowsinsiderpreviewSDK), not sure if that will be the final version number.

Reply:
According to [this article](https://venturebeat.com/2019/11/12/microsoft-starts-rolling-out-windows-10-november-2019-update/) the build number seems to be 18363, but maybe we should wait till the SDK is released.

Reply:
Thanks for looking it up! @ranjeshj will probably have more info about this.

Reply:
19H2 doesn't come with an own SDK. 20H1, however, will (build 19041 or higher).